README.rst

```diff
@@ -18,7 +18,143 @@ SpringField is ideal for:
 - Parsing CSV data structures from ``csv.DictReader``
 - Turning anything Python can parse into a ``dict`` or ``list`` into a structured object
 
+There is also a helper library for using SpringField with Mongo: `springfield-mongo <https://github.com/six8/springfield-mongo>`_
+
+
+Quickstart
+----------
+
+To define an ``Entity``, subclass ``springfield.Entity``. Define your attributes by specifying
+``fields``. This library provides the follow self-describing fields to start with:
+
+- ``IntField``
+- ``FloatField``
+- ``BooleanField``
+- ``StringField``
+- ``BytesField``
+- ``DateTimeField``
+- ``EmailField``
+- ``UrlField``
+- ``EntityField``
+- ``CollectionField``
+
+
+A quick example:
+
+
+::
+
+    #!/usr/bin/env python
+    from springfield import Entity, fields
+    from springfield.timeutil import utcnow
+
+
+    class Bookmark(Entity):
+        uri = fields.UrlField(doc='The bookmark uri.')
+        verified = fields.BooleanField(doc='Whether or not this bookmark URI has been verified to exist.')
+        added = fields.DateTimeField()
+
+
+    class User(Entity):
+        id = fields.IntField(doc='Auto-incremented database id.')
+        email = fields.EmailField(doc='The user\'s email address.')
+        bookmarks = fields.CollectionField(fields.EntityField(Bookmark))
+        created = fields.DateTimeField()
+
+
+    if __name__ == '__main__':
+        user = User()
+        user.id = 5
+        user.email = 'foobar@example.com'
+        user.bookmarks = [
+            {'uri': 'https://github.com'},
+            {'uri': 'ftp://google.com', 'verified': True}
+        ]
+        user.created = utcnow()
+        data = user.to_json()
+        print data
+
+        user = User.from_json(data)
```

Comment:
A comment on what you're doing with JSON would help. It's obvious what's happening, but not why you're doing it.